doc/source/management/data_at_rest_encryption.rst

```diff
@@ -258,22 +258,46 @@ System Variables
   :vartype: Boolean
   :default: ``Off``
 
-When this option is turned on, server starts to encrypt temporary tablespace
-and temporary |InnoDB| file-per-table tablespaces. The option does not force
-encryption of temporary tables which are currently opened, and it doesn't
-rebuild system temporary tablespace to encrypt data which are already written.
+When this option is set to ``ON``, the server encrypts the temporary
```

Comment:
I think it is better to be explicit at the opening line. 

"When this option is set to ON, the server encrypts the global temporary tablespaces (ibtmp* files) and the session temporary tablespaces (#innodb_temp/temp_*.ibt files).

We could remove this duplicated elaborate wording later.